tests/Unit/Product/FeedTest.php

```diff
@@ -1,31 +1,30 @@
 <?php
-namespace Tests;
+namespace BazaarVoice\Product;
 
-use BazaarVoice\ProductFeed;
 use org\bovigo\vfs\vfsStream;
 use org\bovigo\vfs\vfsStreamDirectory;
 use org\bovigo\vfs\vfsStreamWrapper;
 use PHPUnit\Framework\TestCase;
 
-class ProductFeedTest extends TestCase {
+class FeedTest extends TestCase {
 
   public function testNewFeedElement() {
-    $pf = new ProductFeed();
+    $pf = new Feed();
```

Comment:
Keep consistent `Set Expect Actions Assertions` too

Reply:
This tests already exists. Is not the core of this PR, I will make these changes in a separeted PR.